### PR TITLE
JavaScript: Lower precision for a number of queries.

### PR DESCRIPTION
**Pull request by max-schaefer**
These queries are currently run by default, but don't have their results displayed.

Looking through results on LGTM.com, they are either false positives (e.g., `BitwiseSignCheck` which flags many perfectly harmless operations and `CompareIdenticalValues` which mostly flags NaN checks) or harmless results that developers are unlikely to care about (e.g., `EmptyArrayInit` or `MisspelledIdentifier`).

With this PR, the only queries that are still run but not displayed are security queries, where different considerations may apply.

### CODE REVIEW

**Comment by max-schaefer** [comment]
Obviously, feel free to speak up if there are any queries in here that you think are worth keeping.

Two of these queries have associated CWEs; we can keep them on the strength of that, but I think it would be a stretch to call them security queries.

File: change-notes/1.25/analysis-javascript.md [hunk 1]
```diff
@@ -23,6 +23,29 @@
 | Expression has no effect (`js/useless-expression`) | Less results | This query no longer flags an expression when that expression is the only content of the containing file. |
 | Unknown directive (`js/unknown-directive`) | Less results | This query no longer flags directives generated by the Babel compiler. |
 
+The following low-precision queries are no longer run by default on LGTM (their results already were not displayed previously):
```
Comment (asgerf) [inline change-notes/1.25/analysis-javascript.md:26]: Saying both "already" and "previously" seems a bit redundant.

Comment (max-schaefer) [reply change-notes/1.25/analysis-javascript.md]: I have removed "previously".

**Comment by max-schaefer** [comment]
@esbena, can I get a re-approval?

**Pull request merged**